selfdrive/locationd/SConscript

```diff
@@ -1,8 +1,15 @@
-Import('env', 'common', 'cereal', 'messaging')
+Import('env', 'common', 'cereal', 'messaging', 'libkf', 'transformations')
 
 loc_libs = [cereal, messaging, 'zmq', common, 'capnp', 'kj', 'pthread']
 
 env.Program("ubloxd", ["ubloxd.cc", "ublox_msg.cc", "ubloxd_main.cc"], LIBS=loc_libs)
 
 if GetOption("test"):
   env.Program("ubloxd_test", ["ubloxd_test.cc", "ublox_msg.cc", "ubloxd_main.cc"], LIBS=loc_libs)
+
+lenv = env.Clone()
+ekf_sym_cc = env.Object("#rednose/helpers/ekf_sym.cc")
```

Comment:
why not let Program build the object?

Reply:
There was a problem with rednose also using that file, now changed this line to create an .o file with another name, to make sure there are no conflicts